models/ht_neuron.cpp

```diff
@@ -618,21 +661,27 @@ ht_neuron::update( Time const& origin, const long_t from, const long_t to )
         S_.y_ );              // neuron state
 
       if ( status != GSL_SUCCESS )
+      {
         throw GSLSolverFailure( get_name(), status );
+      }
     }
 
     // Deactivate potassium current after spike time have expired
     if ( S_.r_potassium_ && --S_.r_potassium_ == 0 )
+    {
       S_.g_spike_ = false; // Deactivate potassium current.
+    }
 
     // Add new spikes to node state array
     for ( size_t i = 0; i < B_.spike_inputs_.size(); ++i )
+    {
       S_.y_[ 2 + 2 * i ] +=
         V_.cond_steps_[ i ] * B_.spike_inputs_[ i ].get_value( lag );
+    }
```

Comment:
It is really hard to understand the intention of this loop (an update of  DG_AMPA,     DG_NMDA,  DG_GABA_A,  DG_GABA_B state variables). Did you think about unrolling of the loop and the usage of explicit names of state variables would improve comprehensibility of the code?